config/environment.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
-module.exports = function(/* environment, appConfig */) {
-  return { };
+module.exports = function(environment) {
+    var ENV = {
```

Comment:
as you have environment being passed into this exports don't you need to set contentSecurityPolicy on that? I don't know if this is going to affect the consuming app so maybe you need to check if it is already set on environment and append it or something?